src/status_im/subs/chats.cljs

```diff
@@ -358,3 +358,21 @@
    (re-frame/subscribe [:chats/chat chat-id]))
  :->
  :image)
+
+(re-frame/reg-sub
+ :group-chat/manage-members-count
+ :<- [:chats/chats]
```

Comment:
Much preferable to subscribe to `:chats/chat` (singular) because we don't want to subscribe to the top-level `:chats/chats` collection.